website/docs/docs/dbt-versions/product-lifecycles.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "dbt Product lifecycles"
+id: "product-lifecycles"
+description: "Learn about dbt Labs' product lifecycles."
+---
+
+dbt Labs is directly involved with the maintenance of two products: 
+
+- dbt Core: The [open-source](https://github.com/dbt-labs/dbt-core) software that’s freely available 
+- dbt Cloud: The managed [SaaS solution](https://www.getdbt.com/signup) built on top of dbt Core
+
+Any dbt feature will fall into one of the following lifecycle states:
+
+
+### dbt Cloud
+
+- **Beta:** Beta features may be made available for the purpose of customer testing and evaluation. These may not be feature-complete or fully stable. There may still be some planned additions and modifications to product behaviors while in Beta. Breaking changes may occur – although we will do our best to communicate them in advance, we may not always be able to do so. Beta features may not be fully documented, technical support may be limited, and service level objectives (SLOs) may not be provided.
+
+- **Preview (Private or Public):**  Preview features are stable and can be considered for production deployments. There may still be some planned additions and modifications to product behaviors before moving to General Availability. We may also introduce new functionality to Preview features that are not backward compatible. Preview features include documentation, technical support, and include service level objectives (SLOs). Features in Preview are generally provided at no extra cost, although they may become paid features in their Generally Available state.
```

Comment:
Should say: "We may also introduce new functionality to Preview features that **is** not backward compatible"